avocado/utils/multipath.py

```diff
@@ -88,6 +88,81 @@ def get_mpath_name(wwid):
         return process.system_output(cmd, sudo=True).split()[0]
 
 
+def get_mpath_status(mpath):
+    """
+    get the status of mpathX of multipaths
+    :param mpath_name: mpath names. Example: mpatha, mpathb.
+    :return: state of mpathX eg: Active, Suspend, None
+    """
+    cmd = 'multipathd -k"show maps status" | grep -i %s' % mpath
+    mpath_status = process.getoutput(cmd).split()[-2]
+    return mpath_status
+
+
+def suspend_mpath(mpath):
+    """
+    suspending the given mpathX of multipaths
+    :param mpath_name: mpath names. Example: mpatha, mpathb.
+    :return: True or False
+    """
+    def is_mpath_suspended():
+        if get_mpath_status(mpath) == 'suspend':
+            return True
+        return False
+
+    cmd = 'multipathd -k"suspend map %s"' % mpath
+    if process.system(cmd) == 0:
+        return wait.wait_for(is_mpath_suspended, timeout=10) or False
```

Comment:
If `process.system(cmd)` is not equal to `0` this method it will return `None`, right? Is that expected?

I would recommend returning only `True` or `False`.

Reply:
yes, will add the return false too.
Thanks for pointing this out.